### PR TITLE
[MIG] - migrate account_cutoff_accrual_picking into account_cutoff_picking

### PR DESCRIPTION
**Pull request by sbejaoui**
original PR #73 

cc/ @alexis-via , @jbaudoux

### CODE REVIEW

**Comment by rousseldenis** [comment]
@alexis-via @sbejaoui @jbaudoux Could we find an agreement on changes for this one as this make me frighten about mixing implementations? Thanks

File: account_cutoff_picking/__manifest__.py [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+# Copyright 2018-2021 Jacques-Etienne Baudoux (BCIM sprl) <je@bcim.be>
+# Copyright 2013 Alexis de Lattre (Akretion) <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
 
 {
     "name": "Account Cut-off Picking",
-    "version": "16.0.1.2.0",
-    "category": "Accounting",
+    "version": "16.0.1.0.0",
```
Comment (jbaudoux) [inline account_cutoff_picking/__manifest__.py:8]:
```suggestion
    "version": "16.0.2.0.0",
```

File: account_cutoff_picking/models/account_cutoff_line.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+            tax_account_field_name = "account_accrued_expense_id"
+            tax_account_field_label = "Accrued Expense Tax Account"
+        elif self.parent_id.cutoff_type == "accrued_revenue":
+            tax_account_field_name = "account_accrued_revenue_id"
+            tax_account_field_label = "Accrued Revenue Tax Account"
+        for tax_line in tax_info["taxes"]:
+            tax_read = self.env["account.tax"].browse(tax_line["id"])
+            tax_cutoff_account_id = tax_read[tax_account_field_name]
+            if not tax_cutoff_account_id:
+                if not company.cutoff_taxes:
```
Comment (jbaudoux) [inline account_cutoff_picking/models/account_cutoff_line.py:98]:
```suggestion
                if not company.accrual_taxes:
```

**Review by jbaudoux — CHANGES_REQUESTED** [review]

File: account_cutoff_picking/models/account_cutoff_line.py (hunk 2 above)
Comment (jbaudoux) [reply account_cutoff_picking/models/account_cutoff_line.py:98]: missing test coverage on taxes

**Comment by jbaudoux** [comment]
Replaced by https://github.com/OCA/account-closing/pull/268

**Pull request closed without merging**